enrolments/tests/views/test_session.py

```diff
@@ -0,0 +1,74 @@
+from django.urls import reverse
+from rest_framework import status
+from rest_framework.test import APITestCase
+
+from accounts.models import User
+from enrolments.models import Session
+from enrolments.serializers import SessionSerializer
+
+
+class SessionTestCase(APITestCase):
+    def setUp(self):
+        self.user = User.objects.create(email="user@staff.com")
+        self.session_1 = Session.objects.create(season=Session.SPRING, year=2021)
+        self.session_2 = Session.objects.create(season=Session.SUMMER, year=2021)
+
+    def test_get_all_sessions(self):
+        url = reverse("sessions-list")
+        self.client.force_login(self.user)
+        response = self.client.get(url)
+        payload = response.json()
+
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+        self.assertEqual(
+            payload,
+            [
+                SessionSerializer(self.session_1).data,
+                SessionSerializer(self.session_2).data,
+            ],
+        )
+
+    def test_get_session(self):
+        url = reverse("sessions-detail", args=[self.session_1.id])
+        self.client.force_login(self.user)
+        response = self.client.get(url)
+        payload = response.json()
+
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+        self.assertEqual(payload, SessionSerializer(self.session_1).data)
+
+    def test_method_not_allowed(self):
```

Comment:
can we add the POST method in here too?

Reply:
even though we're not sure if there should be a post method yet?

Reply:
whoops should have clarified! I meant we should make sure POST returns a 405 in `test_method_not_allowed` (similar to put and patch)

Reply:
ah ok. Yeah I decided on this test, in case we allow post requests to be made in the near future. But actually keeping it more lean as of now is better